sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Microsoft.Azure.WebPubSub.Common.csproj

```diff
@@ -12,7 +12,6 @@
 
   <ItemGroup>
     <PackageReference Include="System.Text.Json" />
-    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.7.2"/>
```

Comment:
You need just a `<PackageReference Include="System.Text.Encodings.Web"/>` here because you don't depend on Azure.Core. Check your project.assets.json file to be sure after building again.